app/oxcon2023g0/.gitignore

```diff
@@ -0,0 +1,12 @@
+**/*.rs.bk
```

Comment:
Was this committed intentionally?  Should these things be in the top-level `.gitignore`  instead?

Reply:
It's a copy-paste from our other apps, I'm not actually sure who added it or what program generates `.bk` files. Can remove.